api/lib/models/admin/statistics/index.ts

```diff
@@ -0,0 +1,63 @@
+import { EpochNumber, UidType } from '../../../JSCommon/common-types';
+import { IDataMapper, IDbResult } from '../../../services/database';
+import { IUowOpts } from '../../../services/database/svc/uow.service';
+import { AcademicYear, CodingExperience, Gender, ShirtSize, VeteranOptions } from '../../register';
+
+interface IUserCount {
+  pre_count: number;
+  reg_count: number;
+  rsvp_count: number;
+  checkin_count: number;
+}
+
+interface IUserStatistics {
+  // Pre-Registration data
+  pre_uid: UidType;
+  pin: number;
+  // Registration data
+  firstName: string;
+  lastName: string;
+  gender: Gender;
+  shirtSize: ShirtSize;
+  dietaryRestriction: string | null;
+  allergies: string | null;
+  travelReimbursement: boolean;
+  firstHackathon: boolean;
+  university: string;
+  email: string;
+  academicYear: AcademicYear;
+  major: string;
+  phone: string;
+  ethnicity: string | null;
+  resume: string | null;
```

Comment:
Is this supposed to be a string? Because on the api website it says FILE

Reply:
The resume is stored as a URL to the actual location of the file. While on the API we expect a FILE, it is uploaded to GCS and only the URL to it is stored in the db.